README.md

```diff
@@ -51,7 +76,7 @@ Redirect uri must be **http://<edx_url>/auth/complete/custom-oauth2/**
    }
    ```
 
- - If you're want seamless authorization add middleware classes for SeamlessAuthorization (crossdomain cookie support needed)
+ - If you want seamless authorization add middleware classes for SeamlessAuthorization (crossdomain cookie support needed)
    ```
    EXTRA_MIDDLEWARE_CLASSES: ["edx_oauth_client.middleware.SeamlessAuthorization",]
```

Comment:
Please update this to `edx_oauth_client.middleware.seamless_authorization` as the it is now converted to function based middleware.